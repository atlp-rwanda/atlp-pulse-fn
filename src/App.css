@tailwind base;
@tailwind components;
@tailwind utilities;

@layer base {
  li{
    @apply p-4
  }
  button{
    @apply text-white border bg-sky-900 border-sky-400
    rounded-md
  }
}

@media (max-width: 500px) {
    .hideSvg{
        display: none;
    }
}