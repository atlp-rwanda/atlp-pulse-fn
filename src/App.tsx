import React from 'react';

const App = () => {
  // const { data, loading, error } = useQuery(gql`
  //   query shake {
  //     hello
  //   } `)
  // if (loading) return <p>Loading...</p>
  // if (error) return <p>Error: {error.message}</p>
  return <p>This is my first Appollo App</p>;
};

export default App;
