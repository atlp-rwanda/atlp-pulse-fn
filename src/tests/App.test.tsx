import React from 'react';
import renderer from 'react-test-renderer';
import '../../test/jest/__mocks__/matchMedia';
import App from '../App';

describe('App test ', () => {
  it('Should render app', () => {
    const elem = renderer.create(<App />).toJSON();
    expect(elem).toMatchSnapshot();
  });
});
