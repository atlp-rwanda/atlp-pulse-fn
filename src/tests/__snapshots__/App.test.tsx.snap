// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test  Should render app 1`] = `
<p>
  This is my first Appollo App
</p>
`;
